godtools/App/Services/InitialDataDownloader/InitialDeviceResourcesLoader.swift

```diff
@@ -172,17 +170,12 @@ class InitialDeviceResourcesLoader {
         }
     }
     
-    private func setupInitialFavoritedResourcesAndLanguage(complete: @escaping (() -> Void)) {
+    private func setupInitialFavoritedResourcesAndLanguage() {
                 
-        realmDatabase.background { [weak self] (realm: Realm) in
-            
-            self?.favoritedResourcesCache.addToFavorites(realm: realm, resourceId: "2") //satisfied
-            self?.favoritedResourcesCache.addToFavorites(realm: realm, resourceId: "1") //knowing god personally
-            self?.favoritedResourcesCache.addToFavorites(realm: realm, resourceId: "4") //fourlaws
-            self?.favoritedResourcesCache.addToFavorites(realm: realm, resourceId: "8") //teach me to share
-                        
-            complete()
-        }
+        favoritedResourcesCache.addToFavorites(resourceId: "2") //satisfied
+        favoritedResourcesCache.addToFavorites(resourceId: "1") //knowing god personally
+        favoritedResourcesCache.addToFavorites(resourceId: "4") //fourlaws
+        favoritedResourcesCache.addToFavorites(resourceId: "8") //teach me to share
```

Comment:
Hey @rachaelblue would be better not to hard-code those.  Let's leave it for now because this logic is going to go away when this task is implemented -> https://jira.cru.org/browse/GT-1715